pkg/bucketeer/model/evaluation.go

```diff
@@ -8,4 +8,5 @@ type Evaluation struct {
 	VariationID    string  `json:"variationId,omitempty"`
 	Reason         *Reason `json:"reason,omitempty"`
 	VariationValue string  `json:"variationValue,omitempty"`
+	VariationName  string  `json:"variationName,omitempty"`
```

Comment:
Based on the comment below, I added `VariationName` to `Evaluation` to add `VariationName` to `BKTEvaluationDetail`.
https://github.com/bucketeer-io/go-server-sdk/pull/136#discussion_r1686552692